### PR TITLE
WalletController added, LoginController updated

### PR DESCRIPTION
**Pull request by vaishnavirk**
Hey check them and let me know if there are any changes. I haven't added controllers related to categories or services. I will add them later

### CODE REVIEW

File: src/main/java/com/sapient/controllers/WalletController.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+		}
+
+		try {
+			String token = authHeader.split(" ")[1];
+			log.info("token = {}", token);
+			Integer userId = JwtUtil.verify(token);
+
+			Double myBalance = userDao.getBalance(userId);
+			userDao.addToWallet(userId, amount);
+			myBalance = userDao.getBalance(userId);
```
Comment (subSri) [inline src/main/java/com/sapient/controllers/WalletController.java:58]: Multiple declarations at 56 and 58

File: src/main/java/com/sapient/controllers/WalletController.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+		}
+
+		try {
+			String token = authHeader.split(" ")[1];
+			log.info("token = {}", token);
+			Integer userId = JwtUtil.verify(token);
+
+			Double myBalance = userDao.getBalance(userId);
+			userDao.withdrawFromWallet(userId, amount);
+			myBalance = userDao.getBalance(userId);
```
Comment (subSri) [inline src/main/java/com/sapient/controllers/WalletController.java:82]: Same as above

File: src/main/java/com/sapient/controllers/LoginController.java [hunk 3]
```diff
@@ -35,5 +35,17 @@
...
 
 	}
+	
+	@PostMapping("/api/register")
+	public ResponseEntity<?> register(@RequestBody User user) throws Exception {
+		if (!(userDao.verifyUserCreds(user))) {
+			userDao.addNewUser(user);
+			return ResponseEntity.ok("Successfully registered!");
+		} else {
+			return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body("This account already exists!");
```
Comment (subSri) [inline src/main/java/com/sapient/controllers/LoginController.java]: Change Status from unauthorized to StatusCode : 409

**Review by subSri — CHANGES_REQUESTED** [review]
The requested changes have been added as comments

**Review by vaishnavirk — COMMENTED** [review]
HTTP status unauthorised changed to conflict.

**Pull request merged**